app/SectionApp/TodoContainer/Tasks/DeleteStepTask.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\SectionApp\TodoContainer\Tasks;
+
+use App\SectionApp\TodoContainer\Repositories\Interfaces\StepRepositoryInterface;
+use App\SectionApp\TodoContainer\Repositories\Interfaces\TodoRepositoryInterface;
+
+/**
+ * Class DeleteStepTask
+ *
+ * This class is responsible for deleting step.
+ */
+class DeleteStepTask
```

Comment:
Như thế này sẽ clear hơn
$id thì sẽ ko rõ id của todo hay step
```suggestion
/**
 * Class DeleteStepsByTodoIdTask
 *
 * This class is responsible for deleting steps by todo id.
 */
class DeleteStepsByTodoIdTask
```